lib/runner/TestRunner.js

```diff
@@ -300,7 +254,65 @@ module.exports = class TestRunner {
         return results;
     }
 
-    processResponse(response, testSuite, context) {
+
+    async resolveVariablesForAssertionInteraction(assertion, localizedValues, resolveMethod) {
+        if (!assertion._variables || assertion._variables.length === 0) {
+            return localizedValues;
+        }
+
+        for (let j = 0; j < assertion.variables.length; j++) {
```

Comment:
is the same _variables and variables? it would be easy to read if only choose one of them

Reply:
should be _variables for set and variables for get, will add a proper setter to use only variables